assignment-01.md

```diff
@@ -4,27 +4,45 @@ E.g. Question <br>
 {Answer} {Reference}
 
 1. What is the difference between private limited and public limited company? State one example for each type.
+Private Limited Company refers to the company which is not listed on a stock exchange and the shares are held privately by the members concerned while Public Limited Company implies a company that is listed on a recognized stock exchange and whose shares are traded openly by the public.
+Reference: https://keydifferences.com/difference-between-public-company-and-private-company.html
 
 2. Why is Berjaya Sports Toto Berhad registered under Non-Shariah Compliant?
+It is because it is related to gambling which is under Non-Shariah Compliant Business Activities.
 
 3. List down 3 Non-Shariah Compliant Business Activities. 
+Alcohol, Gambling, Pornography 
+Reference: https://av.sc.com/my/content/docs/Non-Syariah-Compliant-Business-Activities_ref.pdf
 
 4. Name the stock exchange of Malaysia.
+Kuala Lumpur Stock Exchange (KLSE)
 
 5. Create a new folder name 'AR' in your _repo main branch_. Upload Annual Report 2018 - 2020 for companies below: <br>
 (a) ViTrox Corp <br>
 (b) Pentamaster Corp <br> 
 *There should be at least 6 PDF files. <br>
```

Comment:
Referring to the instruction above, you are required to **create a new folder in your main branch**. And upload all PDF files there.